newIDE/app/src/Utils/ProjectCache.js

```diff
@@ -3,65 +3,136 @@
 import { serializeToJSON } from './Serializer';
 
 const CLOUD_PROJECT_AUTOSAVE_CACHE_KEY = 'gdevelop-cloud-project-autosave';
+const objectStoreScope = 'cloud-project-autosaves';
+const keyName = 'userProjectKey';
 
 type ProjectCacheKey = {| userId: string, cloudProjectId: string |};
 
 class ProjectCache {
-  cachePromise: Promise<Cache> | null;
+  databasePromise: Promise<IDBDatabase> | null;
 
   static isAvailable() {
-    return typeof window !== 'undefined' && 'caches' in window;
+    return typeof window !== 'undefined' && 'indexedDB' in window;
   }
 
   static async burst() {
     if (!ProjectCache.isAvailable()) return;
-    await caches.delete(CLOUD_PROJECT_AUTOSAVE_CACHE_KEY);
+    return new Promise(resolve => {
```

Comment:
resolve is never called

Reply:
Thanks! I don't know why it was working anyway, maybe because it's called in a `Promise.all`.